docs/getting_started/current-sem-trainings.md

```diff
@@ -24,6 +24,7 @@ Click the name of a training to view more details and register!
 ### **October**
 - [UV Package Manager--Get Your Sunscreen! (an RC Quick Byte)](https://colorado.libcal.com/calendar/events/uv) - **October 1st, 11am, Virtual (Zoom)**
 - [AlphaFold3 on Alpine (RC Short Course)](https://colorado.libcal.com/calendar/events/alphafold3) - **October 7th, 11am, Virtual (Zoom)**
+- [An Introduction to Debugging (an RC Short Course)](https://calendar.colorado.edu/event/an-introduction-to-debugging-an-rc-short-course) - **October 8th, 1pm, Hybrid (Norlin Library E206/Zoom)**
```

Comment:
You are linking to https://calendar.colorado.edu/ when all of the other links go to https://colorado.libcal.com. For consistency sake, please change this.